core/modules/admin_bar/admin_bar.inc

```diff
@@ -802,6 +810,8 @@ function admin_bar_theme_settings() {
   );
 
   $form['components']['admin_bar.icon']['#description'] = t('A specialized menu that links to the home page and provides links to perform various other admin tasks (e.g.: clear caches, run cron, etc.).');
+  $form['components']['admin_bar.back_to_site_link']['#description'] = t
+  ('Changes the "Home" link to a "Back to site" link. It returns to the last non-admin page visited.');
```

Comment:
Alternative text:
`Replaces the "Home" link with a "Back to site" link that returns to the last non-admin page visited.`

Reply:
Yes, this is better.

Reply:
It may have been that at one point.